src/server/worker/datum/datum.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/pps"
 	pfsserver "github.com/pachyderm/pachyderm/v2/src/server/pfs"
 	"github.com/pachyderm/pachyderm/v2/src/server/worker/common"
+	workerStats "github.com/pachyderm/pachyderm/v2/src/server/worker/stats"
```

Comment:
was this just for clarity, or is there another `stats`?

Reply:
It's not a direct naming conflict, but there are stats on the datum object that I didn't want this to be confused with.